src/Module/Profile/Profile.php

```diff
@@ -87,6 +87,7 @@ protected function rawContent(array $request = [])
 					$data = ActivityPub\Transmitter::getProfile($user['uid']);
 					header('Access-Control-Allow-Origin: *');
 					header('Cache-Control: max-age=23200, stale-while-revalidate=23200');
+					header('Vary: Accept', false);
```

Comment:
Please move this line to line 82 which removes the need for the one on line 108.